packages/icons/src/icon-product/air-gapped-inverse.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
is this necessary? not seeing it in the other SVG files

Reply:
With the exception of removing the fill that matched our background color, I basically just used what design gave me but I can adjust.

Reply:
Removed.